### PR TITLE
Interfaces homework

### PR DESCRIPTION
**Pull request by Pokosteler**
Interfaces(homework 7)

### CODE REVIEW

File: src/lesson7/cosmodrome/Shuttle.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package lesson7.cosmodrome;
+
+public class Shuttle implements IStart {
+    @Override
+    public Boolean systemsCheck() {
+        int check = (int) (Math.random() * 10);
+        if (check > 3) {
```
Comment (TheFallen058) [inline src/lesson7/cosmodrome/Shuttle.java]: return check > 3;

File: src/lesson7/cosmodrome/SpaceX.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package lesson7.cosmodrome;
+
+public class SpaceX implements IStart {
+    @Override
+    public Boolean systemsCheck() {
+        int check = (int) (Math.random() * 8 + 4);
+        if (check > 10) {
```
Comment (TheFallen058) [inline src/lesson7/cosmodrome/SpaceX.java]: Check IDE hints:
return check > 10;

File: src/lesson7/robot/Run.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    public static void main(String[] args) {
+        /*
+        Создать по 3 реализации(Sony, Toshiba, Samsung) каждой запчасти(IHead, IHand, ILeg)
+        Класс SonyHead является примером реализацией головы от Sony.
+        Создайте 3 робота с разными комплектующими.
+        Например у робота голова и нога от Sony а, рука от Samsung.
+        У всех роботов вызовите метод action.
+        Среди 3-х роботов найдите самого дорогого.
+        */
+        Robot robot1 = new Robot(new SonyHead(15), new SamsungHand(4), new ToshibaLeg(1));
```
Comment (TheFallen058) [inline src/lesson7/robot/Run.java]: Try to move part price inside part class, so constructor will look like 
Robot firstRobot = new Robot(new SonyHead(), new SamsungHand(), new ToshibaLeg());

File: src/lesson7/robot/Robot.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        leg.step();
+    }
+
+    /**
+     * Get the cost of a lesson7.robot
+     * @return int
+     */
+    @Override
+    public int getPrice() {
+        int price = head.getPrice() + hand.getPrice() + leg.getPrice();
```
Comment (TheFallen058) [inline src/lesson7/robot/Robot.java]: surplus variable
        return head.getPrice() + hand.getPrice() + leg.getPrice();

**Review by TheFallen058 — APPROVED** [review]
Please pay attention to my comments

File: src/lesson7/robot/Robot.java (hunk 4 above)
Comment (Pokosteler) [reply src/lesson7/robot/Robot.java]: fixed

File: src/lesson7/robot/Run.java (hunk 3 above)
Comment (Pokosteler) [reply src/lesson7/robot/Run.java]: fixed

File: src/lesson7/cosmodrome/SpaceX.java (hunk 2 above)
Comment (Pokosteler) [reply src/lesson7/cosmodrome/SpaceX.java]: fixed

File: src/lesson7/cosmodrome/Shuttle.java (hunk 1 above)
Comment (Pokosteler) [reply src/lesson7/cosmodrome/Shuttle.java]: fixed

**Review by TheFallen058 — APPROVED** [review]
Amazing! great work!

**Pull request merged**